walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index-w3s2.html

```diff
@@ -3,14 +3,14 @@
   <head>
     <meta charset="UTF-8">
     <title>Random Quotes</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link rel="stylesheet" href="style-e.css">
+    <script src="script-e.js"></script>
```

Comment:
This looks like an error, you have script.js not script-e.js.

Reply:
Oh I missed this comment. Yes, it is a mistake.